Source/ProgressionSystemRuntime/Private/Data/PSWorldSubsystem.cpp

```diff
@@ -152,6 +157,22 @@ void UPSWorldSubsystem::OnCharacterReady(APlayerCharacter* PlayerCharacter, int3
 	PlayerCharacter->OnPlayerTypeChanged.AddUniqueDynamic(this, &ThisClass::OnPlayerTypeChanged);
 }
 
+//  Is called when a main menu state has been changed
+void UPSWorldSubsystem::OnMainMenuStateChanged(ENMMState NewState)
+{
+	if (NewState == ENMMState::Cinematic)
+	{
+		for (AActor* Actor : SpawnedStarActorsInternal)
+		{
+			if (Actor)
+			{
+				Actor->SetActorHiddenInGame(true);
```

Comment:
From Trello card:

> Expected result:
> Once cinematic started, stars smoothly disappeared: each star scales from (1,1,1) scale to (0,0,0) for X seconds (by default 0.2sec)

It means, you can't do `SetActorHiddenInGame` as it will instantly hide starts right in front of player's view that looks as a bug again.

Remove this code and see following comment, where I explain how to implement animating the star properly